templates/hello-world-ts/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "hello-world-ts",
+  "version": "1.0.0",
+  "description": "nitric typescript hello world template",
+  "main": "index.js",
+  "private": true,
+  "dependencies": {
+    "@nitric/sdk": "^0.3.0"
```

Comment:
Perhaps change the version to "latest" to ensure we don't have to update this again.

Reply:
fixed.